extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -59,6 +68,7 @@ class AmpBaseCarousel extends PreactBaseElement {
       ActionTrust.LOW
     );
     return dict({
+      'advanceCount': advanceCount,
```

Comment:
This should also be (or rather only be) in the `AmpElement.props` declaration, so that the mutations are automatically picked up. I think if we move `next/prev` to component's imperative methods, this will get resolved by itself.

Reply:
Yes, this change was quite natural once we replaced `advance` with `next`/`prev`